util/test/test_query.py

```diff
@@ -735,6 +736,32 @@ def test_populateNftAssetNames():
     assert nfts[0].name == "Take a Ballet Lesson"
 
 
+@enforce_types
+def _test_queryPassiveRewards(addresses):
+    chain = brownie.network.chain
+    S_PER_WEEK = 604800
+    fee_distributor = oceanutil.FeeDistributor()
+    OCEAN = oceanutil.OCEANtoken()
+
+    OCEAN.transfer(
+        fee_distributor.address,
+        toBase18(1000.0),
+        {"from": brownie.accounts[0]},
+    )
+    fee_distributor.checkpoint_token({"from": brownie.accounts[0]})
+    fee_distributor.checkpoint_total_supply({"from": brownie.accounts[0]})
+    chain.sleep(S_PER_WEEK * 3)
```

Comment:
```suggestion
   for i in range(0, 3):
       chain.sleep(S_PER_WEEK)
       chain.mine()
       fee_distributor.checkpoint_token({"from": brownie.accounts[0]})
       fee_distributor.checkpoint_total_supply({"from": brownie.accounts[0]})
```
then do balances, rewards = query.queryPassiveRewards(CHAINID, chain.time(), addresses)

also make sure that lock_time for those addresses is long enough